stable/newrelic-infrastructure/templates/daemonset.yaml

```diff
@@ -32,21 +32,30 @@ spec:
           securityContext:
             privileged: true
           env:
+            {{- if .Values.licenseKey }}
             - name: NRIA_LICENSE_KEY
               valueFrom:
                 secretKeyRef:
                   name: {{ template "newrelic.fullname" . }}-config
                   key: license
+            {{- end }}
+            {{- if and .Values.customSecret (not .Values.licenseKey) }}
+            - name: "NRIA_LICENSE_KEY"
+              valueFrom:
+                secretKeyRef:
+                  name: {{ .Values.customSecret.name }}
+                  key: {{ .Values.customSecret.key }}
+            {{- end }}
             - name: "CLUSTER_NAME"
               value: "{{ .Values.cluster }}"
-           {{- if .Values.kubeStateMetricsUrl }}
+            {{- if .Values.kubeStateMetricsUrl }}
```

Comment:
Where these extra spaces a mistake in in `HEAD`?

Reply:
true